### ISSUE TITLE
Should I Install Chrome or Chromium In AWS For Production On Linux? 

### ISSUE THREAD

**Issue by hafiihzafarhana** [go-rod/rod#1090]
![image](https://github.com/user-attachments/assets/24465bbe-e344-42ae-b6de-fadddbeb8947)

above say can automatically to download or find the browser. Does this feature apply to production stages like on AWS?

Or we should install manual on the AWS Linux?

**Comment by ysmood** [comment]
Yes, as long as your process can access internet. It will download the version specified [here](https://github.com/go-rod/rod/blob/5098fbe03be30abb8cb044b27985633d67775027/lib/launcher/revision.go#L5-L6), you can also config which version to download: 

https://github.com/go-rod/rod/blob/c42b414fffbd412dddac4903e5bc9355c6bb786c/lib/launcher/browser.go#L76-L77

**Comment by hafiihzafarhana** [comment]
How about if I use the Docker image of go-rod (ghcr.io/go-rod/rod:v0.116.2)? is it not enough?

I think Docker image of go-rod includes chrome

https://github.com/go-rod/rod/blob/01a50b925312b71fc59f6eb0d8f489a5195ec448/lib/docker/Dockerfile :

```
RUN ln -s /root/.cache/rod/browser/$(ls /root/.cache/rod/browser)/chrome /usr/bin/chrome
```

**Comment by ysmood** [comment]
Yes, if you use the same version of go-rod image, rod won't download the browser again.
